packages/hebao_v1/contracts/modules/exchanges/LoopringModule.sol

```diff
@@ -110,7 +110,18 @@ contract LoopringModule is SecurityModule
             uint   pubKeyY
         )
     {
-        return exchange.getAccount(wallet);
+        bytes memory callData = abi.encodeWithSelector(
+            exchange.getAccount.selector,
+            wallet
+        );
+        (bool success, bytes memory result) = address(exchange).staticcall(callData);
+        if (success && result.length == 96) {
+            assembly {
+                accountId := mload(result)
+                pubKeyX := mload(add(result, 32))
+                pubKeyY := mload(add(result, 64))
```

Comment:
Needs an additional offset of 32 to skip over the length:
```
accountId := mload(add(result, 32))
pubKeyX := mload(add(result, 64))
pubKeyY := mload(add(result, 96))
```

Reply:
I keep forgetting this